animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -136,18 +152,21 @@ cdef class Mesh:
         self.verticesMoved()
 
     def __repr__(self):
-        return textwrap.dedent("""\
-            AN Mesh Object:
-                Vertices: {}
-                Edges: {}
-                Polygons: {}
-                UV Maps: {}\
-            """.format(len(self.vertices), len(self.edges), len(self.polygons), list(self.uvMaps.keys())))
+        return textwrap.dedent(
+        f"""AN Mesh Object:
+        Vertices: {len(self.vertices)}
+        Edges: {len(self.edges)}
+        Polygons: {len(self.polygons)}
+        UV Maps: {list(self.uvMaps.keys())}
+        Vertex Color Layers: {list(self.vertexColorLayers.keys())}""")
```

Comment:
Can't we use `getVertexColorLayerNames` here?

Reply:
Yes. I have changed it, `Vertex Colors: {self.getVertexColorLayerNames()}`.